packages/tre/src/plugins/cache/index.ts

```diff
@@ -87,7 +82,7 @@ export const CACHE_PLUGIN: Plugin<
               name: BINDING_JSON_CACHE_WARN_USAGE,
               json: JSON.stringify(options.cacheWarnUsage ?? false),
             },
-            loopbackBinding,
+            WORKER_BINDING_SERVICE_LOOPBACK,
           ],
           compatibilityDate: "2022-09-01",
```

Comment:
In the spirit of extracting shared options, the compatibility date could probably also be extracted